lib/util/db.js

```diff
@@ -490,6 +490,11 @@ const postgresErrorToProblem = (x) => {
     if (match != null) {
       return reject(Problem.user.valueTooLong({ maxLength }));
     }
+  } else if (error.code === '23514') {
```

Comment:
what happens / what error gets surfaced if we don't try to catch this specific error like this? do we need to wrap it in a message like this or can we keep it more generic? we might have other integrity constraint violations that now get this message.